README.md

```diff
@@ -72,51 +81,61 @@ rdme openapi [path-to-file.json] --id={existing-id} --version={project-version}
 ```
 
 #### Omitting the File Path
+
 If you run `rdme` within a directory that contains your OpenAPI or Swagger definition, you can omit the file path. We will then look for a file with the following names, and upload that: `openapi.json`, `openapi.yaml`, `swagger.json`, and `swagger.yaml`.
 
 ```sh
 rdme openapi
 ```
 
 #### Validating an API Definition
+
 You can also perform a local validation of your API definition without uploading it to ReadMe, which can be useful when constructing or editing your API definition.
 
 ```sh
 rdme validate [path-to-file.json]
 ```
 
 ### Docs
+
 #### Syncing a Folder of Markdown Docs to ReadMe
+
 ```sh
 rdme docs path-to-markdown-files --version={project-version}
 ```
 
 #### Edit a Single ReadMe Doc on Your Local Machine
+
 ```sh
 rdme docs:edit <slug> --version={project-version}
 ```
 
 ### Versions
+
 #### Get All Versions Associated With Your Project
+
 ```sh
 rdme versions
 ```
 
-If you wish to see the raw JSON output from our API in this response, supply  the `--raw` flag.
```

Comment:
oops

Reply:
Whoa I didn't even catch this, VS Code did 🤯

Reply:
(also it was me that updated this line like a month or two ago, oops)